docker-compose/community-docker-compose.yml

```diff
@@ -81,19 +82,23 @@ services:
       - "5432:5432"
 
   solr6:
-    image: alfresco/alfresco-search-services:2.0.2
+    image: quay.io/alfresco/search-services:2.1.0-SNAPSHOT
     mem_limit: 2g
     environment:
       # Solr needs to know how to register itself with Alfresco
-      - SOLR_ALFRESCO_HOST=alfresco
-      - SOLR_ALFRESCO_PORT=8080
+      SOLR_ALFRESCO_HOST: "alfresco"
+      SOLR_ALFRESCO_PORT: "8080"
       # Alfresco needs to know how to call solr
-      - SOLR_SOLR_HOST=solr6
-      - SOLR_SOLR_PORT=8983
+      SOLR_SOLR_HOST: "solr6"
+      SOLR_SOLR_PORT: "8983"
       # Create the default alfresco and archive cores
-      - SOLR_CREATE_ALFRESCO_DEFAULTS=alfresco,archive
-      # HTTP by default
-      - ALFRESCO_SECURE_COMMS=none
+      SOLR_CREATE_ALFRESCO_DEFAULTS: "alfresco,archive"
+      # HTTPS or SECRET
+      ALFRESCO_SECURE_COMMS: "secret"
+      # SHARED SECRET VALUE
+      SOLR_OPTS: "
```

Comment:
Even though neither SOLR_OPTS / JAVA_TOOL_OPTIONS are perfect, we agreed to go back to recommending JAVA_TOOL_OPTIONS rather than SOLR_OPTS since it has a less broad attack surface compared to SOLR_OPTS. Relevant conversation [here](https://github.com/Alfresco/InsightEngine/pull/1363#discussion_r811271171).

Reply:
Changed to JAVA_TOOL_OPTIONS.